### ISSUE TITLE
Legend Text Location

### ISSUE THREAD

**Issue by DanielHurtubise** [BadMonkeysTeam/Mandrill#54]
Is there any way to control the Legend location to have it on top
![image](https://user-images.githubusercontent.com/4299930/50136141-1197f780-0297-11e9-815f-ce5edb7cfcc1.png)
Thanks

**Comment by ksobon** [comment]
I don't think so. What I would do, is just feed in empty string for that legend text, and then add a text note above that legend. That should get you what you need.
